ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+TabObserver.swift

```diff
@@ -53,16 +53,16 @@ extension BrowserViewController: TabObserver {
       injectedScripts.append(contentBlocker)
     }
 
-    #if canImport(BraveTalk)
-    injectedScripts.append(
-      BraveTalkScriptHandler(
-        rewards: rewards,
-        launchNativeBraveTalk: { [weak self] tab, room, token in
-          self?.launchNativeBraveTalk(tab: tab, room: room, token: token)
-        }
+    if braveTalkJitsiCoordinator.isIntegrationEnabled {
```

Comment:
This should be swapped with `prefService.isBraveTalkAvailable` as it also handles `braveRequestAdsEnabled` checks